platforms/openal/SoundSystemAL.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <thirdparty/OpenAL/Include/al.h>
+#include <thirdparty/OpenAL/Include/alc.h>
+
+#include <string>
+#include <vector>
+#include <unordered_map>
+
+#include "client/sound/SoundSystem.hpp"
+
+#pragma comment( lib, "OpenAl32.lib" )
```

Comment:
This should be in a `#ifdef _WIN32` block.

Reply:
Is there anything needed for other platforms to include the libraries?

Reply:
That's what [this piece of CMake](https://github.com/ReMinecraftPE/mcpe/blob/14562fc8cef55954681c6d1c591922ca81931fdf/source/CMakeLists.txt#L249-L255) does. On that note, `USE_OPENAL` has to be added to https://github.com/ReMinecraftPE/mcpe/blob/14562fc8cef55954681c6d1c591922ca81931fdf/platforms/sdl/CMakeLists.txt#L5.